roles/network_plugin/calico/templates/calico-node.yml.j2

```diff
@@ -168,6 +168,9 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+            # Set AS number
+            - name: AS
```

Comment:
Regarding https://docs.projectcalico.org/v3.5/reference/node/configuration, setting the AS number here would prevent per-node peerings (with ASnumbers different from the global one) from working.